src/patterns/OcFileActions.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div uk-offcanvas="mode: slide" class="uk-offcanvas uk-offcanvas-bottom">
+    <div class="uk-offcanvas-bar">
+      <span v-text="_label"></span>
+      <ul class="uk-nav">
+        <slot />
+      </ul>
+    </div>
+  </div>
+</template>
+
+<style>
+.uk-offcanvas.uk-offcanvas-bottom {
```

Comment:
we need to pull this into the less/scss files - maybe use proxy classes .... @felixheidecke